src/models/parallelCoordinates.js

```diff
@@ -58,32 +58,44 @@ nv.models.parallelCoordinates = function() {
             var gEnter = wrapEnter.append('g');
             var g = wrap.select('g');
 
-            gEnter.append('g').attr('class', 'nv-parallelCoordinatesWrap');
+            gEnter.append('g').attr('class', 'background');
+            gEnter.append('g').attr('class', 'foreground');
+
             wrap.attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
 
-            var line = d3.svg.line(),
-                axis = d3.svg.axis().orient('left'),
-                background,
-                foreground;
+            var line = d3.svg.line().interpolate('cardinal').tension(lineTension),
+                axis = d3.svg.axis().orient('left');
```

Comment:
It's used line 110

 dimension.append('g')
                .attr('class', 'axis')
                .each(function(d) { d3.select(this).call(axis.scale(y[d])); })